lib/authentication/view/edit_profile_page.dart

```diff
@@ -22,10 +24,95 @@ class EditProfilePage extends StatefulWidget {
+    final oldPasswordController = TextEditingController();
+    final changePasswordKey = GlobalKey<FormState>();
+    return AppDialog(
+      title: S.of(context).actionChangePassword,
+      content: [
+        Form(
+          key: changePasswordKey,
+          child: Column(
+            children: [
+              TextFormField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    labelText: S.of(context).labelOldPassword,
+                    hintText: S.of(context).hintPassword,
+                  ),
+                  controller: oldPasswordController,
+                  validator: (value) {
+                    if (value.isEmpty || value == null) {
+                      return S.of(context).errorNoPassword;
+                    }
+                    return null;
+                  }),
+              TextFormField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    labelText: S.of(context).labelNewPassword,
+                    hintText: S.of(context).hintPassword,
+                  ),
+                  controller: newPasswordController,
+                  validator: (value) {
+                    if (value.isEmpty || value == null) {
+                      return S.of(context).errorNoPassword;
+                    }
+                    return null;
+                  }),
+              TextFormField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    labelText: S.of(context).labelConfirmNewPassword,
+                    hintText: S.of(context).hintPassword,
+                  ),
+                  validator: (value) {
+                    if (value.isEmpty || value == null) {
+                      return S.of(context).errorNoPassword;
+                    }
+                    if (value != newPasswordController.text) {
+                      return S.of(context).errorPasswordsDiffer;
+                    }
+                    return null;
+                  }),
+            ],
+          ),
+        )
+      ],
+      actions: [
+        AppButton(
+          key: ValueKey('change_password_button'),
+          text: S.of(context).actionChangePassword.toUpperCase(),
+          color: Colors.lightBlue,
+          width: 130,
+          onTap: () async {
+            if (changePasswordKey.currentState.validate()) {
+              AuthProvider authProvider =
+                  Provider.of<AuthProvider>(context, listen: false);
+              if (await authProvider.verifyPassword(
+                  password: oldPasswordController.text, context: context)) {
+                if (await AppValidator.isStrongPassword(
+                    password: newPasswordController.text, context: context)) {
+                  bool res = await authProvider.changePassword(
+                      password: newPasswordController.text, context: context);
+                  if (res) {
```

Comment:
Nit: The inconsistency is bugging me
```suggestion
                  if (await authProvider.changePassword(
                      password: newPasswordController.text, context: context)) {
```